command/smartContract.go

```diff
@@ -0,0 +1,55 @@
+package command
+
+import (
+	"github.com/rubixchain/rubixgoplatform/client"
+	"github.com/rubixchain/rubixgoplatform/core"
+)
+
+func (cmd *Command) generateSmartContractToken() {
+	smartContractTokenRequest := core.GenerateSmartContractRequest{
+		BinaryCode: cmd.binaryCodePath,
+		RawCode:    cmd.rawCodePath,
+		SchemaCode: cmd.schemaFilePath,
+		DID:        cmd.did,
+	}
+
+	request := client.SmartContractRequest{
```

Comment:
This can be updated as follows

request := client.SmartContractRequest{
		BinaryCode: cmd.binaryCodePath,
		RawCode:    cmd.rawCodePath,
		SchemaCode: cmd.schemaFilePath,
		DID:        cmd.did,
	}